source/types/ut_test.tpb

```diff
@@ -87,16 +83,17 @@ create or replace type body ut_test is
         -- most likely occured in setup or teardown if here.
         ut_assert.report_error(sqlerrm(sqlcode) || ' ' || dbms_utility.format_error_stack);
         ut_assert.report_error(sqlerrm(sqlcode) || ' ' || dbms_utility.format_error_backtrace);
+
+        self.execution_result := ut_assert.get_test_results;
```

Comment:
This overwrite would only happen if flow would get into the exception block due to exception after line 70.
There is nothing that could cause that - no overwrite.